protos/flyteidl/core/tasks.proto

```diff
@@ -11,6 +11,17 @@ package flyteidl.core;
 
 option go_package = "github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/core";
 
+// Metadata associated with the GPU accelerator to allocate to a task. Contains
+// information about device type, and for multi-instance GPUs, the partition size to
+// use.
+message GPUAccelerator {
+    string device = 1;
```

Comment:
Nit: could we add a comment or example (e.g. `A100`, `Nvidia/GPU`, something like that) for device?